start.sh

```diff
@@ -1,13 +1,15 @@
 #! /bin/bash
 
 # Reset iC880a PIN
-gpio -1 mode 22 out
-gpio -1 write 22 0
-sleep 0.1
-gpio -1 write 22 1
-sleep 0.1
-gpio -1 write 22 0
-sleep 0.1
+pin=25
```

Comment:
could you please use all upper case for the variable and a more descriptive name, like `SX1301_RESET_BCM_PIN` (25 is 22 in BCM2835 denomination, right?)